Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+	  <LangVersion>latest</LangVersion>
+	  <ImplicitUsings>enable</ImplicitUsings>
+	  <IncludeSource>true</IncludeSource>
+	  <AssemblyName>Frends.SFTP.UploadFiles</AssemblyName>
+	  <RootNamespace>Frends.SFTP.UploadFiles</RootNamespace>
+
+	  <Version>1.0.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
Lets stick to our MIT license really and then lets have a separate apache license file called like "Apache whatever.txt" and in our README we can mention the fact that we are using serilog and such. Also, make sure to include the apache license in the Nuget package.

Reply:
Fixed